third_party/VENDOR-LICENSE

```diff
@@ -1711,37 +1711,6 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 
 
-===========================================================
```

Comment:
why does this part need to be deleted?

Reply:
I've been seeing this too, I'd bet it's messed up at HEAD.